044-cluster-wide-volume-usage-quota-management.md

```diff
@@ -0,0 +1,330 @@
+# Cluster Wide Volume Usage Quota Management
+
+Extend the static quota mechanism to throttle message production in a broker if any active broker in the cluster is running out of disk.
+
+Deprecate the current implementation that considers only aggregate local volume usage.
```

Comment:
Not deprecated anymore ...

Reply:
Removed this line, I think the removal is covered enough in the other sections.